constitution.md

```diff
@@ -0,0 +1,684 @@
+
+_MemGen-2_: A person who is not an individual is not eligible to be a member of the association.
+
+### _MemApp_: Membership applications
+
+_MemApp-1_: An application by a person to be a member of the association must be:
+
+- made in writing, and
+- in the form determined by the committee, and
+- lodged with the secretary.
+
+_MemApp-2_: The committee may determine that an application may be made or lodged by email or other electronic means.
+
+_MemApp-3_: The secretary must determine whether to approve or reject the application or refer the nomination to the committee which is to determine whether to approve or to reject the nomination.
+
+_MemApp-4_: The committee must approve or reject the application.
+
+_MemApp-5_: As soon as practicable after the committee has decided the application, the secretary must:
+
+- _MemApp-5-Not_: give the applicant written notice of the decision, including by email or other electronic means if determined by the committee, and
+- _MemApp-5-App_: if the application is approved - inform the applicant that the applicant is required to pay the entrance fee and annual subscription fee payable under clause _MemFee_ within 28 days of the day the applicant received the notice.
+
+_MemApp-6_: The secretary must enter the applicant’s name in the register of members as soon as practicable after the applicant pays the entrance fee and annual subscription fee in accordance with subclause _MemReg_.
+
+_MemApp-7_: The applicant becomes a member once the applicant’s name is entered in the register.
+
+### _MemReg_: Register of members
+
+_MemReg-1_: The secretary must establish and maintain a register of members of the association.
+
+_MemReg-2_: The register:
+
+* may be in written or electronic form, and
+* must include, for each member:
+  - the member’s full name, and
+  - a residential, postal or email address, and
+  - the date on which the person became a member, and
+  - if the person ceases to be a member - the date on which the person ceased to be a member, and
+* _MemReg-2-NSW_: if not elecronic must be kept in Australia:
+  - at the association’s main premises, or
+  - if the association has no premises - at the association’s official address, and
+* _MemReg-2-Ins_: must be available for inspection, free of charge, by members at a reasonable time.
+
+_MemReg-3_: If the register is kept in electronic form, the requirements in subclause _MemReg-2-NSW_ and _#MemReg-2-Ins_ apply as if a reference to the register is a reference to a current hard copy of the register.
+
+_MemReg-4_: A member may obtain a hard copy of the register, or a part of the register, on payment of a fee of not more than $1, as determined by the committee, for each page copied.
+
+_MemReg-5_: No information about a member, other than the member’s name can be made available for inspection unless the member authorises that information to be released.
```

Comment:
❓ How does this interact with MemReg-2-Ins, since the register contains additional information and must be made available for inspection?

Reply:
I confess I don't know.  The clause in the model constitution is: _MemReg-5: Information about a member, other than the member’s name, must not be made available for inspection if the member requests that the information not be made available._  I changed it from opt out to opt in.  I presume the original was prepared by lawyers and it worked, so this must work too.